.travis.yml

```diff
@@ -23,10 +22,11 @@ script:
 
 deploy:
   provider: releases
-  api_key: "$GH_TOKEN"
+  token: "$GH_TOKEN"
+  file: ./*.jar
```

Comment:
Doesn't look like the file is found on deploy when I ran the build. Needs more testing.